<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="IND-48NG453" failures="2" tests="7" name="Test" time="0.04" errors="0" timestamp="2021-03-14T21:09:42 IST">
  <testcase classname="TestCases.validation" name="validationTest" time="0.008">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Not NULL expected object to not be null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Not NULL expected object to not be null
at TestCases.validation.validationTest(validation.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- validationTest -->
  <testcase classname="TestCases.test2" name="LogOff" time="0.001"/>
  <testcase classname="TestCases.test2" name="Login" time="0.0"/>
  <testcase classname="TestCases.test2" name="ValidateUser" time="0.0"/>
  <testcase classname="TestCases.test1" name="LogOff" time="0.001">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestCases.test1.LogOff(test1.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- LogOff -->
  <testcase classname="TestCases.test1" name="Login" time="0.0"/>
  <testcase classname="TestCases.test1" name="ValidateUser" time="0.002"/>
</testsuite> <!-- Test -->
